classes/02class/agile101/README.md

```diff
@@ -0,0 +1,326 @@
+# Class 02
+
+This class is an introduction to Agile methodology and frameworks.
+
+The main goal is to explain the difference between sequential life cycle software development and iterative ones. 
+
+There is also a brief introduction to Scrum and Kanban frameworks.
```

Comment:
This could reside under the Agile Topic.

Reply:
This is about the class goal, a class introduction, not Agile specific